plugins/reports/frontend/public/templates/vue-main.html

```diff
@@ -19,16 +19,43 @@ <h2> {{i18n('reports.title')}}
 
 
 <script  type="text/x-template" id="reports-table">
-<cly-section>
+<div>
+<div v-if="tableRows.length === 0" >
+    <div class="bu-is-flex  bu-is-flex-direction-column bu-is-align-items-center empty-view">
+        <div class="no-content-circle">
+        </div>
+        <div   style="width: 400px;">
+            <h3 class="color-cool-gray-100 bu-mt-5 bu-mb-4 bu-has-text-centered">...hmm, seems it’s empty here</h3>
+            <div class="color-cool-gray-50 text-medium bu-has-text-centered">
+                Create something to Lorem ipsum dolor sit amet,
+                consectetur adipiscing elit, 
+            </div>
+            <div  v-on:click="createReport()" class="button bu-has-text-centered bu-mt-5 color-blue-100 pointer">+ Create New Report</div>
+        </div>
+    </div>
+</div>
```

Comment:
I believe you are creating a global component for the empty state. Right? @waiterZen 
Will you add that to this pr only ? Or should I merge this pr and later you will change this code and use the global component here ?

Reply:
global component also here now.